gt/main.c

```diff
@@ -1691,10 +1691,9 @@ gt_proc(void *arg)
 		 * XXX #71 Do something better here!
 		 * For now, free any unsent packets.
 		 */
-		if (unlikely(num_tx_succ < num_tx)) {
-			for (i = num_tx_succ; i < num_tx; i++)
-				rte_pktmbuf_free(tx_bufs[i]);
-		}
+		if (unlikely(num_tx_succ < num_tx))
+			rte_pktmbuf_free_bulk(&tx_bufs[num_tx_succ],
+				num_tx - num_tx_succ);
```

Comment:
Style. Given that you had to break the call `rte_pktmbuf_free_bulk()` into two lines, bring back the braces of the `if` statement.